exporter/README.md

```diff
@@ -0,0 +1,251 @@
+# WAL-G Prometheus Exporter
+
+A Prometheus exporter for WAL-G backup and WAL metrics for PostgreSQL databases.
+
+## Features
+
+- **Backup lag monitoring**: Track time since last backup-push (full and delta backups)
+- **WAL lag monitoring**: Monitor time since last wal-push operation
+- **LSN delta lag**: Calculate LSN lag in bytes between current and archived WAL
+- **PITR window**: Monitor point-in-time recovery window size
+- **Error monitoring**: Track WAL-G operation errors
+- **WAL integrity**: Monitor WAL segment integrity status per timeline
+
+## Metrics
+
+The exporter provides the following metrics:
+
+### Backup Metrics
+- `walg_backup_lag_seconds{backup_type}` - Time since last backup-push in seconds
```

Comment:
for all timestamps, let's specify clearly if it's timestamp of beginning of the process of end of it